test/WebJobs.Extensions.Tests/Extensions/NotificationHub/ConverterTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.NotificationHubs;
+using Microsoft.Azure.WebJobs.Extensions.NotificationHubs;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Tests.Extensions.NotificationHub
+{
+    public class ConverterTests
+    {
+        [Fact]
+        public void Converter_JsonString_Valid()
+        {
+            Converter converter = new Converter();
+            string messageProperties = "{\"message\":\"Hello from Node! \",\"location\":\"Redmond\"}";
+            TemplateNotification templateNotification = converter.BuildTemplateNotificationFromJsonString(messageProperties);
+            Assert.NotNull(templateNotification);
```

Comment:
You should also assert that the properties inside the TemplateNotification are what you expect -- same for the other tests. Just in case some parsing goes wrong somewhere.